### DISCUSSION TITLE
Can't stay logged in?

### DISCUSSION THREAD

**Discussion by OBVYamaska** [opengisch/QField#2604] [Q&A]
Hi!
I'm beginning to test QfieldCloud: awesome work!
Most of it work well so far, but one thing that keeps coming up is login prompts.

I would get a prompt to login again on almost every action I'm taking, but on a random basis.
Most of them are not a real problem, but I had these even when syncing projects from or to the Cloud, which caused the download to finish incomplete.
Of course, I checked the box to remember my credentials ;)

**Comment by suricactus** [comment]
@OBVYamaska what QFieldSync version are you using? There was a problem a few BETAs ago. Can you try installing the latest 4.0.0 and confirm this is still an issue? Also QGIS and operating system information would be very helpful.

**Comment by OBVYamaska** [comment]
Sorry about that!
I use QGIS 3.22.4, and had installed the latest QFieldSync as of yesterday, so [4.0.0-BETA20]. I just saw the "no-Beta" v4, which I just installed. I'll report when I have a chance to test it.

**Reply by suricactus** [reply]
Operating system?

**Reply by OBVYamaska** [reply]
Windows 10 Pro (21H2). Sorry for the delay, I'm experiencing weird behavior preventing me to comment and respond normally...

**Reply by suricactus** [reply]
we all did, GitHub engineers screwed their service for a few minutes :) . Thanks!

**Comment by OBVYamaska** [comment]
I tested the plugin yesterday with the new version (v4 non-Beta) and everything went smoothly!
I'll keep the post open today and test some more, but I'll probably close it after that. Thanks for the fix!
